.storybook/addons/codepreview/SyntaxHighlighter.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from 'react';
+import { styled } from '@storybook/theming';
+
+import { ActionBar } from '@storybook/components/dist/ActionBar/ActionBar';
+
+import ReactSyntaxHighlighter from 'react-syntax-highlighter';
+
+const Wrapper = styled.div(
+  ({ theme }) => ({
+    position: 'relative',
+    overflow: 'hidden',
+    color: theme.color.defaultText,
+  }),
+  ({ theme, bordered }) =>
+    bordered
+      ? {
+          border: `1px solid ${theme.appBorderColor}`,
+          borderRadius: theme.borderRadius,
+          background: theme.background.content,
+        }
+      : {},
+);
+
+const Pre = styled.pre(({ theme, padded }) => ({
+  display: 'flex !important',
+  justifyContent: 'flex-start',
+  margin: 0,
+  padding: padded ? `${theme.layoutMargin}px !important` : 0,
+  tabSize: '2',
+}));
+
+const Code = styled.code`
+  flex: 1;
+  padding-right: 0;
+  opacity: 1;
+  counter-reset: line;
+
+  .code-line {
```

Comment:
I somehow don't like that we use plain classes like this and nothing bem-ified. Evern in storybook, this could lead to conflicts at some ponits. We could probably use the stories / addon-names as "block".